README.md

```diff
@@ -11,7 +11,7 @@ node ./installdroid.js (for Android)
 
 ## Make sure to run the react-native packager
 ```shell
-npm run-script start (Windows users please use "npm run-script start-windows" instead of npm run-script start)
+npm start (Windows users please use "npm run-script start-windows" instead of npm run-script start)
```

Comment:
You should also fix the npm run-script start in within the parentheses
npm start (Windows users please use "npm run-script start-windows" instead of **npm start**)